inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -31,12 +31,18 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*Reduce.*type=Logical.*)",
         R"(.*Reduce.*axes=\(1\.-1\).*)",
         R"(.*Reduce.*axes=\(0\.3\)_type=Prod.*)",
-        // TODO: Issue 34055
+        // TODO: Issue: 34518
         R"(.*RangeLayerTest.*)",
+        R"(.*(RangeAddSubgraphTest).*Start=1.2.*Stop=5.2.*Step=0.1.*netPRC=FP16.*)",
+        // TODO: Issue: 34083
 #if (defined(_WIN32) || defined(_WIN64))
         R"(.*(CoreThreadingTestsWithIterations).*(smoke_LoadNetworkAccuracy).*(targetDevice=HETERO).*)",
 #endif
         // TODO: Issue: 34348
         R"(.*IEClassGetAvailableDevices.*)",
+        // TODO: Issue: 25533
+        R"(.*ConvertLikeLayerTest.*)",
```

Comment:
Doesn't important for the PR but just a general comment - from regex perspective braces here (and almost everywhere in existing skips) are redundant